.github/workflows/vm-kernel.yaml

```diff
@@ -293,12 +309,26 @@ jobs:
           TAG: ${{ needs.get-tags.outputs.canonical }}
 
         run: |
-          result_image="${VM_KERNEL_IMAGE}:${TAG}"
+          result_image="${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}:${TAG}"
 
           args=("-t" "${result_image}")
           for arch in ${ARCHS}; do
-            args+=("${VM_KERNEL_IMAGE}-${arch}:${TAG}")
+            args+=("${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}-${arch}:${TAG}")
           done
           docker buildx imagetools create "${args[@]}"
 
           echo "image=${result_image}" >> $GITHUB_OUTPUT
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
+
+      - name: Copy vm-kernel to Docker Hub
+        env:
+          TAG: ${{ needs.get-tags.outputs.canonical }}
+        run: |
+          echo Copy ${VM_KERNEL_IMAGE}:${TAG} to Docker Hub
+          docker buildx imagetools create -t "${DOCKERHUB_REGISTRY}/${VM_KERNEL_IMAGE}:${TAG}" \
+                                             "${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}:${TAG}"
```

Comment:
I see that vertical alignment by image name makes such strings more readable, similar to what you did here:
https://github.com/neondatabase/autoscaling/pull/1283/files#diff-6ec9d6ffc2c8ebd369edcc698ad16020762b6616d45849d4b539557e59563884R403

wdyt to have same approach here?

Reply:
Good idea, applied!